### PR TITLE
add extra cols for gui and version in MM2.db table

### PR DESCRIPTION
**Pull request by smk762**
- Adds extra columns to MM2.db `stats_swaps` table. Ref: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2061
- Updates DB folder location. Ref: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2102

### CODE REVIEW

File: src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx [hunk 1]
```diff
@@ -3,7 +3,20 @@ export const description = "This guide describes how to query the MM2 SQLite dat
...
+```
+30 08:39:23, mm2:269] AtomicDEX API 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, mm2_main::mm2::lp_native_dex:500] INFO Version: 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, crypto::crypto_ctx:324] INFO Public key hash: 7d6cbdd91788df3b764247721fe12853ce36b03d
+```
+
+The location of this folder for each operating system is:
+- Linux: $HOME/.kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
+- MacOS: $HOME/Library/Application Support/kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
+- Windows: %APPDATA%\kdf\DB\{'{wallet identifying hex string}'\MM2.db'
```
Comment (onur-ozkan) [inline src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx:18]: Why is `.kdf` used for Linux and `kdf` used for other systems?

Comment (smk762) [reply src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx]: The `.` prefix in linux hides the folder and is commonly used for user data folders.

File: src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx [hunk 2]
```diff
@@ -3,7 +3,22 @@ export const description = "This guide describes how to query the MM2 SQLite dat
...
+30 08:39:23, mm2:269] AtomicDEX API 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, mm2_main::mm2::lp_native_dex:500] INFO Version: 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, crypto::crypto_ctx:324] INFO Public key hash: 7d6cbdd91788df3b764247721fe12853ce36b03d
+```
+
+The location of this folder for each operating system is:
+
+```bash
+- Linux: $HOME/.kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
+- MacOS: $HOME/Library/Application Support/kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
```
Comment (laruh) [inline src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx:19]: On MacOs by default the DB path is the same as on Linux
Screenshot 2024-09-13 at 18 13 25

Also please add doc description somewhere, that if you dont want DB to be generated in home/.kdf you need to provide path in `conf["dbdir"]` param

**Review by laruh — CHANGES_REQUESTED** [review]

File: src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx (hunk 2 above)
Comment (smk762) [reply src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx:19]: fixed

File: src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx [hunk 3]
```diff
@@ -3,7 +3,26 @@ export const description = "This guide describes how to query the MM2 SQLite dat
...
+30 08:39:23, mm2:269] AtomicDEX API 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, mm2_main::mm2::lp_native_dex:500] INFO Version: 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, crypto::crypto_ctx:324] INFO Public key hash: 7d6cbdd91788df3b764247721fe12853ce36b03d
+```
+
+The default location of this folder for each operating system is:
+
+```bash
+- Linux: $HOME/.kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
+- MacOS: $HOME/Library/Application Support/kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
```
Comment (laruh) [inline src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx:19]: there is still issue with default kdf location on MacOs. Its not in Application Support, it is located in the home directory 
This is how DB path looks like on my machine
`/Users/sharon/.kdf/DB`

**Review by laruh — CHANGES_REQUESTED** [review]

**Review by AndrewDelaney — APPROVED** [review]

File: src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx (hunk 3 above)
Comment (smk762) [reply src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx:19]: @AndrewDelaney @ShantanuSharma9873 can you please confirm the default location on windows?

**Review by laruh — APPROVED** [review]
LGTM!

**Review by ShantanuSharma9873 — APPROVED** [review]

**Review by gcharang — APPROVED** [review]

**Pull request merged**